Makefile

```diff
@@ -70,22 +70,28 @@ start-kvstores:
         -initial-cluster-token etcd-cluster-1 \
         -initial-cluster-state new
 	-$(DOCKER) rm -f "cilium-consul-test-container" 2> /dev/null
+	rm -rf /tmp/cilium-consul-certs
+	mkdir /tmp/cilium-consul-certs
+	cp $(CURDIR)/test/consul/* /tmp/cilium-consul-certs
 	$(DOCKER) run -d \
            --name "cilium-consul-test-container" \
-           -p 8501:8500 \
+           -p 8501:8443 \
            -e 'CONSUL_LOCAL_CONFIG={"skip_leave_on_interrupt": true, "disable_update_check": true}' \
+	   -v /tmp/cilium-consul-certs:/cilium-consul/ \
            consul:1.1.0 \
-           agent -client=0.0.0.0 -server -bootstrap-expect 1
+	   agent -client=0.0.0.0 -server -bootstrap-expect 1 -config-file=/cilium-consul/consul-config.json
```

Comment:
Please fix indentation here. We use only tabs in Makefiles.

I know that's not your mistakem and it seems that indentation was broken here before.